neo4j/internal/collections/set_test.go

```diff
@@ -147,6 +147,28 @@ func TestSet(outer *testing.T) {
 			t.Error(err)
 		}
 	})
+
+	outer.Run("contains", func(t *testing.T) {
```

Comment:
nit: I think this one is covered by the first test above, see:
```
outer.Run("adds", func(t *testing.T) {
// [...]
```

Reply:
Good spot - I like how this one explicitly tests that the set contains a value using the public Contains function rather than relying on the "adds" to capture it but happy to remove either way.